<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maditor::View::Dialogs::NewClassDialog</class>
 <widget class="QDialog" name="Maditor::View::Dialogs::NewClassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameField"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="typeField">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <item>
        <property name="text">
         <string>Entity-Component</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Game-Handler</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Global API</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gui-Handler</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other Class</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scene-Component</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="alreadyExistsMsg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string>Class already exists</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Maditor::View::Dialogs::NewClassDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Maditor::View::Dialogs::NewClassDialog</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Maditor::View::Dialogs::NewClassDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nameChanged(QString)</slot>
 </slots>
</ui>
